metadata-ingestion/tests/unit/test_schema_util.py

```diff
@@ -73,7 +73,7 @@ def assert_field_paths_are_unique(fields: List[SchemaField]) -> None:
         assert len(avro_fields_paths) == len(set(avro_fields_paths))
 
 
-def assret_field_paths_match(
+def assert_field_paths_match(
```

Comment:
Thanks for the contrib!
This function is used in a few places, so you will have to do a find/replace of this in all callers.

Reply:
Ah sorry my bad, this was a super drive by 1am edit on github.com, should have realized it was a util function not a `def test_...` function used once 🤦. Should be fixed in 65410437e.

Reply:
thank you!